.circleci/generate_config_yml.py

```diff
@@ -91,14 +90,24 @@ def gen_build_workflows_tree():
         cimodel.data.simple.bazel_definitions.get_workflow_jobs,
         caffe2_build_definitions.get_workflow_jobs,
         cimodel.data.simple.binary_smoketest.get_workflow_jobs,
-        binary_build_definitions.get_binary_smoke_test_jobs,
-        binary_build_definitions.get_binary_build_jobs,
         cimodel.data.simple.nightly_ios.get_workflow_jobs,
         cimodel.data.simple.nightly_android.get_workflow_jobs,
+        windows_build_definitions.get_windows_workflows,
+        binary_build_definitions.get_binary_smoke_test_jobs,
```

Comment:
not moving the smoke tests?

Reply:
smoke tests are actually a part of a separate workflow that only gets run on `postnightly` activites, in the long run it'd be good to have them run immediately after nightlies get uploaded, but that's not entirely the case right now.

In a follow up to this PR I'd like to make it so that update-s3-html jobs are dependent on the upload jobs.

This would make getting these postnightly jobs into the `binary_builds` workflow a lot easier.